arangod/Aql/grammar.hpp

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.7.2.  */
+/* A Bison parser, made by GNU Bison 3.7.1.  */
```

Comment:
will this cause any problems?(different version used as file which is already checked in)

Reply:
Good Catch!
AFAIK we rebuilt the generated files once before every official release to avoid potential issues.
We just check them in to be able to build everything if Bison/Flex are not installed, or outdated on the developers machine / You build without MaintainerMode. (Maybe also for Windows, but I am not sure about this)

So no I think in practice this does not matter.